### DISCUSSION TITLE
LSP (rust-analyzer) errors notification when failing to load workspace

### DISCUSSION THREAD

**Discussion by edevil** [LazyVim/LazyVim#4424] [Q&A]
Hey folks.

In a Rust project, when I define a module in Cargo.toml using a wrong path, for example, Rust Analyzer fails to load the workspace.

In VsCode there is a button/notification at the bottom and when we click we see relevant RA logs.

![Screenshot 2024-09-10 at 15 31 57](https://github.com/user-attachments/assets/5d33e085-8f0b-492f-8bb2-e7a7580b4a8b)

Is there an equivalent way in neovim/lazynvim to get this notification?

Thanks.

**Comment by dpetka2001** [comment]
Most probably `:LspLog` to see the lsp logs. Not aware if there's another way to do it.

**Reply by edevil** [reply]
If we were to implement this feature of surfacing LSP errors, what would be the module to do it in?
